src/utils/map-utils.ts

```diff
@@ -1,7 +1,7 @@
 import { Map as MapBoxMap } from 'mapbox-gl';
 import { LayerKey } from '../config/types';
 
-export function isMapBoundaryAvailable(
+export function isLayerOnView(
   map: MapBoxMap | undefined,
   boundaryId: LayerKey,
```

Comment:
Let's either augment this function to catch other layers and rename boundaryId to layerId, not just lines

Or add a comment explaining that this only works for boundaries at the moment. Indeed, we are currently checking for `${boundaryId}-line`

Reply:
@erwagasore - has eric's comment above been addressed?

Reply:
Yes.